test/support/case.ex

```diff
@@ -0,0 +1,30 @@
+# Copyright 2023 SECO Mind Srl
+# SPDX-License-Identifier: Apache-2.0
+
+defmodule EdgehogDeviceForwarder.Case do
```

Comment:
`Case` is too generic, usually you have stuff like `DataCase`, `ConnectionCase` etc. Think about what this case is covering specifically and name it like that.